sources/packages/web/src/router/StudentRoutes.ts

```diff
@@ -9,6 +9,7 @@ import SelectProgram from "../views/student/financial-aid-application/SelectProg
 import FinancialInfo from "../views/student/financial-aid-application/FinancialInfo.vue";
 import ConfirmSubmission from "../views/student/financial-aid-application/ConfirmSubmission.vue";
 import { StudentRoutesConst } from "../constants/routes/RouteConstants";
+import { AppConfigService } from "@/services/AppConfigService";
```

Comment:
Use of @  - i know its a vue3 thing but not a standard for us. And it may some day interfere with jest